Libraries/LibJS/Tests/numeric-literals-basic.js

```diff
@@ -0,0 +1,22 @@
+function assert(x) { if (!x) throw 1; }
+
+try {
```

Comment:
```suggestion
try {
```

(https://github.com/SerenityOS/serenity/commit/bc40908d32934908ca76686c4288faec0a50c240)

Reply:
done!